advisories/2.3.0/BRSA-tfps4zoe7xte.toml

```diff
@@ -0,0 +1,17 @@
+[advisory]
+id = "BRSA-tfps4zoe7xte"
+title = "kernel CVE-2024-41009"
+cve = "CVE-2024-41009"
+severity = "moderate"
+description = "In the Linux kernel, the following vulnerability has been resolved: -  - bpf: Fix overrunning reservations in ringbuf"
```

Comment:
nit: I'm not sure I like the string of characters here `resolved: -  - bpf: Fix`. Is there a rationale for it? If it renders a certain way, that would be good to call out but this feels like bad input into the advisory and I'd like to trim this if we can?

Reply:
This is straight from the ALAS wesbite:  https://alas.aws.amazon.com/cve/json/v1/CVE-2024-41009.json

If you'd like, I'm okay with omitting either all of `In the Linux kernel, the following vulnerability has been resolved:` for advisories, or reducing the `-` and `:`. I'm not exactly sure where the newline `\n` is converted to `-`, must be in the conversion to TOML format, but another option could be to remove the `-` entirely? This was rendering the same for our GHSAs when we started pulling wording for kernel advisories directly from ALAS: https://github.com/bottlerocket-os/bottlerocket/security/advisories/GHSA-mw65-44mw-v359